routers/user/home.go

```diff
@@ -281,46 +290,41 @@ func Issues(ctx *context.Context) {
 
 	showReposMap := make(map[int64]*models.Repository, len(counts))
 	for repoID := range counts {
```

Comment:
should this`repoID` be `repoIDs`?

Reply:
no - this is a for loop wich do for each repoID ...

Reply:
ah of course. I missed that, thanks :)